packages/gui/package.json

```diff
@@ -2,13 +2,95 @@
   "name": "@chia/gui",
   "author": "Chia Network <hello@chia.net> (https://chia.net/)",
   "description": "GUI for Chia Blockchain",
+  "license": "Apache-2.0",
   "productName": "Chia Blockchain",
   "private": true,
   "homepage": "./",
   "main": "build/electron/main.js",
   "engines": {
     "node": ">=12.20.0"
   },
+  "build": {
+    "productName": "chia",
+    "appId": "net.chia.blockchain",
+    "asar": true,
+    "asarUnpack": "**/daemon/**",
+    "npmRebuild": false,
+    "files": [
+      "build",
+      "daemon",
+      "!node_modules",
+      "!src",
+      "!public"
+    ],
+    "fileAssociations": [
+      {
+        "ext": "offer",
+        "name": "Chia Offer",
+        "description": "Chia Offer",
+        "role": "Editor"
+      }
+    ],
```

Comment:
Can we remove this for now? I know it was in my earlier PR, but we're not quite ready to expose this functionality.

Reply:
I removed the lines from `packages/gui/package.json`